src/lib/components/event/event-summary-row.svelte

```diff
@@ -324,21 +322,33 @@
   >
     <div class="flex items-center gap-2">
       {#if pendingAttempt}
-        <Badge class="mr-1" type={pendingAttempt > 1 ? 'danger' : 'default'}>
+        <Badge
+          class="mr-1"
+          type={hasPendingActivity.paused
+            ? 'warning'
+            : pendingAttempt > 1
+              ? 'danger'
+              : 'default'}
+        >
           <Icon
-            class="mr-1 inline {pendingAttempt > 1 && 'font-bold text-red-400'}"
-            name="retry"
+            class="mr-1 inline {pendingAttempt > 1 &&
+              'font-bold text-red-400'} {hasPendingActivity.paused &&
+              'font-bold text-yellow-700'}"
```

Comment:
Can you extract these into a `$derived` for code cleanliness?

Reply:
How do you feel about using tailwind merge?

```js
        class={merge(
            'mr-1 inline',
            pendingAttempt > 1 && 'font-bold text-red-400',
            hasPendingActivity.paused && 'font-bold text-yellow-700',
        )}
 ```

Reply:
👍